glean/metrics.yaml

```diff
@@ -1538,4 +1538,26 @@ sample:
           The source of the device removal information.
           Either 'TaskRemoveDevice' or 'PushMessage'.
         type: string
-# End of push messages validation metrics
+  # End of push messages validation metrics
+
+  perf_time_to_main_screen:
+    type: event
+    lifetime: ping
+    send_in_pings:
+      - main
+    description: |
+      The time the app took between starting and rendering the main screen
```

Comment:
I think we would need to instrument this on all pages, not just the main page. That bug seemes to happen in random pages at random times.

Reply:
While I agree in general, I'm concerned that generating events for every navigation is more on the creepy scale. Given we know this also happens on load, I would prefer we have one probe and see what we are working with.

Reply:
Hm, I am not sure I would agree it's "creepy"... I guess since we are using events only here that might feel that way because it kinda feel like following the user everywhere, however we do have a bunch of "open this" "click that" events that are basically exactly that already. 

Well, no strong opinions. We can go with just this for and see what we can get out of it at analysis time.